src/Raven.Server/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -72,11 +72,14 @@ public static void WriteSubscriptionTaskConnectionPerformanceStats(this Abstract
             writer.WritePropertyName(nameof(connectionStats.Script));
             writer.WriteString(connectionStats.Script);
             writer.WriteComma();
-            
-            writer.WritePropertyName(nameof(connectionStats.Exception));
-            writer.WriteString(connectionStats.Exception);
-            writer.WriteComma();
-            
+
+            if (connectionStats.Exception != null)
```

Comment:
I think we should add this propertly even when it is null. 

{

} 

is different than:

{ 
   "Exception": null
}

Reply:
done